doc.go

```diff
@@ -24,7 +24,13 @@ Use the Open() function to create a database handle with connection parameters:
 
 	db, err := sql.Open("snowflake", "<connection string>")
 
-The Go Snowflake Driver supports the following connection syntaxes (or data source name (DSN) formats):
+The Go Snowflake Driver supports the following connection syntaxes (or data source name (DSN) formats).
```

Comment:
In general, it might be better to use the placeholder `<account_identifier>` with a link to https://docs.snowflake.com/en/user-guide/admin-account-identifier.html

Rather than having copies of this information in several different places, we should rely on https://docs.snowflake.com/en/user-guide/admin-account-identifier.html to cover most of the information and then provide a couple of examples to illustrate what that means in the context of the Go driver.

Reply:
Yes! I need a link to the admin-account-identifier page. Done. Thank you.